.github/workflows/Perf_Env_Build_Test_CI.yml

```diff
@@ -70,7 +70,7 @@ jobs:
         run: |
           # Install Dockerfile content for pytest
           # install oc/kubectl
-          oc_version="4.13.0-0.okd-2023-06-04-080300"
+          oc_version="4.13.0-0.okd-2023-09-03-082426"
```

Comment:
For the future, it would be really good to parameterize this somewhere so you don't need to modify so many places.  Possibly this should be a secret or otherwise something in the environment.  But it is important to have a PR when these change, so we can determine when things change.

Reply:
We can put it in secret but it will not be visible for us, do you think its ok ?

Reply:
A secret's not really right here because we can't determine when it changes.  Can we template these versions?

Reply:
I'm just not familiar with how github actions work to know whether we can provide template variables in a command line or something.